views/partials/nhl_player_card

```diff
@@ -0,0 +1,41 @@
+
+<script id="card" type="text/x-handlebars-template">
```

Comment:
it's not done yet. some items wil likely change. I thoguth it was worth it for easier organizing even if not quite as modular.